configs/setr/setr_naive_vit-large_8x1_768x768_80k_cityscapes.py

```diff
@@ -0,0 +1,68 @@
+_base_ = [
```

Comment:
rename `configs/setr/setr_naive_vit-large_8x1_768x768_80k_cityscapes.py` to `configs/setr/setr_vit-large_naive_8x1_768x768_80k_cityscapes.py`